File: check_shelters.js
const fs = require('fs');

const data = JSON.parse(fs.readFileSync('shelters_parsed.json', 'utf8'));

const matches = [];
data.forEach((item, index) => {
  const str = JSON.stringify(item);
  if (str.includes('樂業') || str.includes('60號') || (item.lat && Math.abs(item.lat - 24.138048) < 0.005)) {
    matches.push({ index, item });
  }
});

console.log('Total matching items in shelters_parsed.json:', matches.length);
matches.forEach(m => console.log(`Index ${m.index}:`, m.item));

File: check_shelters.py
import json

with open('shelters_parsed.json', 'r', encoding='utf-8', errors='ignore') as f:
  data = json.load(f)

matches = []
for i, item in enumerate(data):
  item_str = json.dumps(item, ensure_ascii=False)
  if '樂業' in item_str or '60號' in item_str or '24.138' in item_str:
    matches.append((i, item))

print(f"Total matching items in shelters_parsed.json: {len(matches)}")
for i, m in matches:
  print(f"Index {i}: {m}")
